File: frontend/src/app/dto/user.dto.ts
export class User {
    username:string;
    password:string;
    name:string;
    surname:string;
    email:string;
    phoneNumber:string;
    organizationName:string;

    constructor (
        username:string,
        password:string,
        name:string,
        surname:string,
        email:string,
        phoneNumber:string,
        organizationName:string
    ) {
        this.username = username;
        this.password = password;
        this.name = name;
        this.surname = surname;
        this.email = email;
        this.phoneNumber = phoneNumber;
        this.organizationName = organizationName;
    }
}

File: backend/mycrm/src/main/java/parrino/riccardo/mycrm/dto/UserDTO.java
package parrino.riccardo.mycrm.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class UserDTO {
    private String username;
    private String password;
    private String name;
    private String surname;
    private String email;
    private String phoneNumber;
    private String organizationName;
}
